### ISSUE TITLE
iOS - Send message - "You appear to be offline.'' message is shown twice in IOS in New Chat Page

### ISSUE THREAD

**Issue by izarutskaya** [Expensify/App#28326]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Login on the IOS app with a new account
2. go to settings > preferences > and check force offline
3. click on the + floating action button
4. click send message
5. observe the ''You appear to be offline'' message in the New Chat Page

## Expected Result:
'You appear to be offline.'' message is shown once in IOS in New Chat Page

## Actual Result:
''You appear to be offline.'' message is shown twice in IOS in New Chat Page

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [x] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 

**Reproducible in staging?:** Y

**Reproducible in production?:** Y

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

![Capture d’écran 2023-09-26 121605](https://github.com/Expensify/App/assets/115492554/47ce2db3-bf23-41d3-898f-97c121422f73)

![IMG_0967](https://github.com/Expensify/App/assets/115492554/c96ccf8d-678f-4e4f-bfcc-44f8fea9de97)

**Expensify/Expensify Issue URL:**

**Issue reported by:** @rayane-djouah

**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1695728283307099

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by izarutskaya** [comment]
## Proposal from @rayane-djouah

### Please re-state the problem that we are trying to solve in this issue.
''You appear to be offline.'' message is shown twice in New Chat Page
### What is the root cause of that problem?
the ScreenWrapper in NewChatPage component is showing a second offline message in addition to the existing one of the high level component.
### What changes do you think we should make in order to solve the problem?
we should add `shouldShowOfflineIndicator={false}` for the ScreenWrapper in NewChatPage

https://github.com/Expensify/App/blob/cfad43ff96f2a104c51fd475e07f0980c4379fcd/src/pages/NewChatPage.js#L171-L177

Result:
https://github.com/Expensify/App/assets/77965000/adae6aa1-1cad-4546-a197-d07dd119d2d8

### What alternative solutions did you explore? (Optional)

**Comment by zanyrenney** [comment]
Hmm, no i can't reproduce this on staging in my account. I see the OP says. "new account" can we confirm its only new accounts?

**Comment by zanyrenney** [comment]
please can you add a screen recording with the existing account? @rayane-djouah

**Comment by zanyrenney** [comment]
Great, thanks for confirming the behaviour there. @rushatgabhane any ideas on if the proposal above will work? Thanks!

**Comment by zanyrenney** [comment]
bump @rushatgabhane can you review the proposal?

**Comment by rojiphil** [comment]
> Have confirmed can't reproduce on iphone and @kbecciv also confirmed in the thread, closing!

@zanyrenney @kbecciv 
Can you please reconfirm reproducing this bug? 
The reason for my request is that I am still able to reproduce this bug. 
Please refer to the attached video that reproduces this issue for me.

https://github.com/Expensify/App/assets/3004659/2dfabc5e-7748-4b67-8201-5f5939d43cf2

**Comment by rojiphil** [comment]
@zanyrenney 
I fail to understand why this issue is closed as I can still reproduce this in v 1.3.91-6

Here is a test video that reproduces the problem:

https://github.com/Expensify/App/assets/3004659/e7a6b4f9-71a4-4bce-a916-5dc826b480e4

**Comment by rojiphil** [comment]
> Maybe we can reopen and fix this for $125 as it's super minor?

I can understand price being reduced initially itself. Or I can even understand price being reduced before receiving any proposals. 
As I understand, sometimes the fix is simple and sometimes it takes much more time. So, overall, it get’s balanced. But, this does not mean that we can ignore the time spent to investigate.
To me, defining an issue super minor after receiving proposals does not sound fair.
Do we have any parallel to substantiate this? 
Maybe, I am missing something here.

**Comment by situchan** [comment]
> Do we have any parallel to substantiate the previous comment?

Example: https://github.com/Expensify/App/issues/29459#issuecomment-1776935863

**Comment by rojiphil** [comment]
> Example: [#29459 (comment)](https://github.com/Expensify/App/issues/29459#issuecomment-1776935863), though it end up being [complex](https://github.com/Expensify/App/pull/30240#issuecomment-1783001636)

Yes. Agree that it is complex. But, I doubt if we can draw a parallel with that issue.
I have had experiences where the issue was closed by stating it as a non-issue even after I sent proposals. But, then, I can understand.
Anyway, to me, this issue happens in a typical flow of starting a new chat. And, I think, we cannot ignore it.
Looks like the confusion is in connection with reproducing this issue.
